api/api.go

```diff
@@ -2,14 +2,16 @@ package api
 
 import (
 	"encoding/json"
-	"gopkg.in/cas.v2"
+	gc "gopkg.in/cas.v2"
```

Comment:
I don't like that this package needs to import the CAS package, but I'm not sure what the ideal way to do it is at the moment... let's talk tomorrow